docs/wiki/research/eODS.md

```diff
@@ -0,0 +1,277 @@
+# enshrined OPERATOR-DELEGATOR SEPARATION (eODS)
+> [!WARNING]
+> This document covers an active area of research, may be outdated at time of reading and subject to future updates, as the design space around Validator roles unbundling evolves.
+## Roadmap tracker
+
+| Upgrade |    URGE     |       Track       |                    Item                    |         Cross-references         |
+| :-----: | :---------: | :---------------: | :----------------------------------------: | :------------------------------: |
+|  eODS   | the Scourge | Staking economics | Solutions to liquid staking centralization | SSF, IL, ePBS, ET, Light Clients |
+## Relevant context 
+Principal–Agent problem, in which the interests of the Agent are not aligned with the interests of the Principal, is part of any delegation, and even more so present in today's staking scene[^1].
+
+Regarding the staking economy, specific market structures have naturally developed, after the Merge, inherited from the perennial distinction between labor and capital.
+Thus, staking has split naturally in two classes of participants, outside protocol level[^2]:
+
+|    Tier    |                                                        Current natural separation                                                        | Slashing risk |
+| :--------: | :--------------------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+| Delegators |   ETH stakers, with no minimum commitment and no strict requirement to participate in any other way beyond bringing in their principal   |   Slashable   |
+| Operators  | Professional node operators providing sophisticated services, with their reputation or some fixed amount of capital of their own at risk |   Slashable   |
```

Comment:
Need not be professional, could be some solo operators too (e.g., Rocket Pool)

Reply:
changed it to : _Node operators providing node running services, with their reputation or some fixed amount of capital of their own at risk_